internal/witness/cmd/witness/example_config.yaml

```diff
@@ -1,3 +1,57 @@
 Logs: 
```

Comment:
Bonus point for a follow-up PR that adds a script / bash 1-liner to re-generate this config from the log list :-)

Reply:
Don't worry, it's coming!

Reply:
🎉